cmd/porter/bundle.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"github.com/deislabs/porter/pkg/porter"
+	"github.com/spf13/cobra"
+)
+
+func buildBundleCommands(p *porter.Porter) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "bundle",
+		Short: "bundle commands",
+	}
+
+	cmd.AddCommand(buildBundleInstallCommand(p))
+
+	return cmd
+}
+
+func buildBundleInstallCommand(p *porter.Porter) *cobra.Command {
+	opts := porter.InstallOptions{}
+	cmd := &cobra.Command{
+		Use:   "install",
+		Short: "Install a bundle",
+		Example: `  porter install
+  porter install --insecure
+  porter install --file myapp/bundle.json
+  porter install --name MyAppInDev
+  porter install --param-file base-values.txt --param-file dev-values.txt --param test-mode=true --param header-color=blue
+  porter install --cred azure --cred kubernetes
+`,
+		PreRunE: func(cmd *cobra.Command, args []string) error {
+			return opts.Prepare()
+		},
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return p.InstallBundle(opts)
+		},
+	}
+
+	f := cmd.Flags()
+	f.BoolVar(&opts.Insecure, "insecure", false,
+		"Allow installing untrusted bundles")
+	f.StringVarP(&opts.File, "file", "f", "bundle.json",
+		"Path to the CNAB definition to install")
+	f.StringVar(&opts.Name, "name", "",
+		"Name of the claim, defaults to the name of the bundle")
+	f.StringSliceVar(&opts.ParamFiles, "param-file", nil,
+		"Path to a parameters definition file for the bundle, each line in the form of NAME=VALUE. May be specified multiple times.")
+	f.StringSliceVar(&opts.RawParams, "param", nil,
+		"Define an individual parameter in the form NAME=VALUE. Overrides parameters set with the same name using --param-file. May be specified multiple times.")
+	f.StringSliceVarP(&opts.CredentialSets, "cred", "c", nil, "Credential to use when installing the bundle. May be either a named set of credentials or a filepath.")
```

Comment:
Worth mentioning that this can be specified multiple times, as the two above?

Reply:
Oh yes! I want to call that out for anything that repeatable since most people don't know how to figure that out from "stringArray" or whatever cobra prints out in the help text.

Thanks, I'll fix that right now.